.env.development

```diff
@@ -0,0 +1 @@
+VUE_APP_API_DOMAIN = "http://localhost:8081"
```

Comment:
how do you select whether your hitting production versus dev?

Reply:
the build and serve scripts set it, but you can override in dev to hit production if needed